code/game/turfs/open/floor/fancy_floor.dm

```diff
@@ -123,14 +123,12 @@
 	var/turfverb = "uproot"
 	tiled_dirt = FALSE
 	max_integrity = 80
-	icon_x_offset = 9
-	icon_y_offset = 9
+	transform = MAP_SWITCH(TRANSLATE_MATRIX(9, 9), matrix())
 
 /turf/open/floor/grass/no_border
 	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN)
 	canSmoothWith = list()
-	icon_x_offset = 0
-	icon_y_offset = 0
+	transform = MAP_SWITCH(TRANSLATE_MATRIX(0, 0), matrix())
```

Comment:
this should just be `matrix()`. `TRANSLATE_MATRIX(0,0)` is just unnecessary processing. same for the rest of them